lib/adal.js

```diff
@@ -713,6 +750,11 @@ AuthenticationContext.prototype.saveTokenFromHash = function (requestInfo) {
             this._saveItem(this.CONSTANTS.STORAGE.ERROR_DESCRIPTION, 'Invalid_state. state: ' + requestInfo.stateResponse);
         }
     }
+
+    if (window.parent && window.parent.loadframeStatus && resource) {
```

Comment:
For detecting iframe, the check should be `window !== window.parent`. I know we have a similar check in handleWindowCallback method, that needs to be updated too.